rali/rali_pybind/example/tf_mnistTrainingExample/README.md

```diff
@@ -0,0 +1,35 @@
+## Tensorflow MNIST digit classification example with RALI pipeline
+
+## Prerequisites
+
+* [ROCm supported hardware](https://rocm.github.io/ROCmInstall.html#hardware-support) 
+	* AMD Radeon GPU or APU required
```

Comment:
Codacy found an issue: [[list-item-content-indent] Don’t use mixed indentation for children, add 1 space](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6366586)